index.html

```diff
@@ -827,34 +837,35 @@ <h4>Detecting when the next media frame will be rendered</h4>
       <h2>Recommendations</h2>
       <p>
         This section describes recommendations from the Media &amp; Entertainment
-        Interest Group for the development of a generic <a>media timed event</a> API,
-        and associated synchronization considerations.
+        Interest Group for the development of a generic <a>timed metadata</a>
+        API, and associated synchronization considerations.
       </p>
       <section>
-        <h3>Subscribing to event streams</h3>
+        <h3>Subscribing to receive timed metadata cues</h3>
         <p>
           The API should allow web applications to subscribe to receive specific
-          event streams by event type. For example, to support MPEG-DASH
+          schemas of <a>timed metadata</a> cue. For example, to support MPEG-DASH
           <code>emsg</code> and MPD events, the API should allow subscription by
           <code>id</code> and (optional) <code>value</code>. This is to make
-          receiving events opt-in from the application point of view. The user
-          agent should deliver only those events to a web application for which
-          the application has subscribed. The API should also allow web
-          applications to unsubscribe from specific event streams by event type.
+          receiving the cues opt-in from the application's point of view. The
+          user agent should deliver only those cues to a web application for
+          which the application has subscribed. The API should also allow web
+          applications to unsubscribe from specific schemas.
         </p>
       </section>
       <section>
-        <h3>Out-of-band events</h3>
-        <p>
-          To be able to handle out of band events, including MPEG-DASH MPD
-          events, the API should allow web applications to create events to be
-          added to the <a>media timeline</a>, to be triggered by the user agent.
-          The API should allow the web application to provide all necessary
-          parameters to define the event, including start and end times, event
-          type, and data payload. The payload should be any data type (e.g., the
-          set of types supported by the WebKit <code>DataCue</code>). For
-          MPEG-DASH MPD events, the event type is defined by the <code>id</code>
-          and (optional) <code>value</code> fields.
+        <h3>Out-of-band timed metadata cue delivery</h3>
+        <p>
+          To be able to handle <a>out-of-band</a> <a>timed metadata</a>,
+          including MPEG-DASH MPD events, the API should allow web applications
+          to create <a>timed metadata</a> cues to be added to the <a>media
+          timeline</a>, to be triggered by the user agent. The API should allow
+          the web application to provide all necessary parameters to define the
+          cue, including start and end times, schema identifier, and data
+          payload. The payload should be any data type (e.g., the set of types
```

Comment:
Do we need to say anything about payload type signalling?

Reply:
Is this covered by "schema identifier"?

Reply:
Not obviously - that sounds like it's something that would help to _parse_ the payload data of the cue, not to _select_ it.

Reply:
suggest s/schema identifier/schema identifier, type identifier

Reply:
The latest update has "cue type identifier", in both the "Subscribing to receive media timed event cues" and "Out-of-band events" sections.